.circleci/config.yml

```
@@ -49,6 +49,7 @@ commands:
             cp -r script/config/examples/tls/sentinel1 ./sentinel1_tls
             cp -r script/config/examples/tls/sentinel2 ./sentinel2_tls
             cp -r script/config/examples/tls/sentinel3 ./sentinel3_tls
+            mkdir -m 777 ./run
```

Why is this 777?

---

Not that it matters much but maybe better
```suggestion
            mkdir -m 1777 ./run
```